dist/jquery.flot.js

```diff
@@ -2162,11 +2184,11 @@ Licensed under the MIT license.
                 ++dec;
             }
 
-            return dec;
+            return isFinite(dec) ? dec : 0;
```

Comment:
The precision could give us Infinity for example if our axis.max is greater than 1e308. In this case, make it 0.